packages/kbn-i18n/README.md

```diff
@@ -244,6 +246,62 @@ class RootComponent extends Component {
 }
 ```
 
+Optionally we can pass `context` prop into `FormattedMessage` component.
+This prop is optional context comment that will be extracted by i18n tools
+and added as a comment next to translation message at `defaultMessages.json`
+
+
+#### Attributes translation in React
+React wrapper provides an API to inject the imperative formatting API into a React
+component via its `props`. This should be used when your React component needs to
+format data to a string value where a React element is not suitable; e.g., a `title`
+or `aria` attribute, or for side-effect in `componentDidMount`. Another approach
+is to use render callback pattern by wrapping your components into `I18nContext`
```

Comment:
What's the utility in having two different ways to do this?

Reply:
Just for convenience, I can get rid of `injectIntl` wrapper if it looks redundant

Reply:
Yeah, we can always add it in later if people want it, but we might as well start with everyone being consistent.